extensions/kafka-connect/src/main/java/com/hazelcast/jet/kafka/connect/KafkaConnectSources.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.kafka.connect;
+
+import com.hazelcast.internal.util.Preconditions;
+import com.hazelcast.jet.kafka.connect.impl.KafkaConnectSource;
+import com.hazelcast.jet.pipeline.SourceBuilder;
+import com.hazelcast.jet.pipeline.StreamSource;
+import org.apache.kafka.connect.source.SourceRecord;
+
+import javax.annotation.Nonnull;
+import java.net.URL;
+import java.util.Properties;
+
+import static com.hazelcast.internal.util.Preconditions.checkRequiredProperty;
+
+/**
+ * Contains factory methods to create a Kafka Connect source.
+ */
+public final class KafkaConnectSources {
```

Comment:
Is this going to be production ready in 5.3? I would rather be safe and used `@Beta` annotation on the public APIs.

Reply:
added: https://github.com/hazelcast/hazelcast/pull/23312/commits/1280e90509436ce201667dd156d5399f2f24972b